Assets/Scripts/Content.cs

```diff
@@ -81,7 +82,7 @@ void NextLine()
 
     public void SentenceBehaviour()
     {
-        if (Input.GetMouseButtonDown(0) || Input.GetButtonDown("Add1"))
+        if (Input.GetMouseButtonDown(0))
```

Comment:
You'll need to add Input.GetButtonDown("Add1") back into the if statement for the controller to work